src/Apps/Artist/routes.tsx

```diff
@@ -17,7 +17,7 @@ import { ShowsRouteFragmentContainer as ShowsRoute } from "./Routes/Shows"
 // * `Redirect` needs to be casted, as it’s not compatible with `RouteConfig`
 export const routes: RouteConfig[] = [
   {
-    path: "/artist/:artist_id",
+    path: "/artist/:artistID",
```

Comment:
@mzikherman - When i was building the new filter I discovered that this has to be `artist_id` as the form of the variable is passed along to the query made to gravity / elastic search.

Reply:
Yea, I think the code mods in V2 may have made that all work, this seemed to be fine. Everything, included the filter , is nested under `artist`. And the filter should automatically get the `artist_id` param added from the parent by the resolver.

Reply:
Cool, just want to confirm. Before it was `artistID`, but with the new dynamicism of the rebulit artwork filter some of the values being passed to the backend are implicit, and `found` will take the variable form from the route defintion (`:artist_id`) and pass it directly over. Worth double checking that everything works as expected and if so 👍